src/menu.c

```diff
@@ -37,7 +37,7 @@ void show_menu() {
 void parse_input() {
 	char input = 0;
 	while (1) {
-		printf(">");
+		printf("\n>");
```

Comment:
Beim Testen der Eingabe ist mir aufgefallen, dass das Prompt symbol immer nach den letzten ausgegebenen Zeichen stand. Daher ein kleines zusatz mit \n

Reply:
Oh, interessant, bei mir war das irgendwie nicht so sondern "schön" in der nä. Zeile: whatever, passt!